src/explore-education-statistics-admin/src/pages/release/data/components/ReleaseDataUploadsSection.tsx

```diff
@@ -120,13 +120,13 @@ const ReleaseDataUploadsSection = ({
 
       if (values.uploadType === 'csv') {
         file = await releaseDataFileService.uploadDataFiles(releaseId, {
-          name: values.subjectTitle,
+          name: values.subjectTitle.trim(),
```

Comment:
I _think_ there's a `trim` method on `Yup.string`, so it would be good to do this validation client-side before sending these requests off